jaiqu/jaiqu.py

```diff
@@ -6,15 +6,15 @@
 from .helpers import identify_key, create_jq_string, repair_query, dict_to_jq_filter
 
 
-def validate_schema(input_json: dict, output_schema: dict, key_hints=None) -> tuple[dict, bool]:
+def validate_schema(input_json: dict, output_schema: dict, open_ai_api_key: str | None = None, key_hints=None) -> tuple[dict, bool]:
```

Comment:
inconsistent with how you did the other ones

Reply:
this is the one is the only one exposed to the developer, the others were internal